src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import hamburgerIcon from "../../assets/navbar/hamburger.svg";
+import logoPrimary from "../../assets/navbar/logoPrimary.svg";
+import logoSecondary from "../../assets/navbar/logoSecondary.svg";
+import Page from "./Page";
+import SideBarWrapper from "./SideBarWrapper";
+
+const Navbar = () => {
+  const [toggle, setToggle] = useState(false);
+  const page: Page = Page.MAIN;
```

Comment:
as states in https://github.com/isd-sgcu/johnjud-frontend/pull/8/files#r1436868806